pkg/media/h264reader/h264reader_test.go

```diff
@@ -0,0 +1,34 @@
+package h264reader
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestH264Reader(t *testing.T) {
```

Comment:
I wouldn't even use real H264. I would just do something like this. 

```
testH264 := []byte{0x00, 0x00, 0x00, 0x01, 0xAB, 0xAB}
```

Then we can do tests for
* Does it handle the different start codes
* How does it handle invalid data